lib/main.dart

```diff
@@ -13,10 +14,11 @@ class MyApp extends StatelessWidget {
     return MaterialApp(
       title: 'Flutter Demo',
       theme: ThemeData(
+        scaffoldBackgroundColor: const Color.fromARGB(255, 77, 117, 79),
 
         useMaterial3: true,
       ),
-      home: const MyHomePage(title: 'Flutter Demo Home Page'),
+      home: const ResultsPage(), 
```

Comment:
are we leaving it like this for testing purposes?

Reply:
yeah can we change this back?